### PR TITLE
Update to elastic 7.0 (6.0)

### LINKED ISSUE #3: Update for beats v6 compatibility
Beats has changed a bit since the original implementation, would be great to see the blog post updated to be relevant to beats v6 and corresponding code updated for compatibility with the latest release.

Also since this https://github.com/elastic/kibana/pull/6796 now allows sub-second aggs, perhaps remove the workaround from the original beat implementation (zoom field).

### PR DESCRIPTION
**Pull request by dadoonet**
Closes #3

### CODE REVIEW

File: data/meta.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"uuid":"4e872c12-2c8a-42b4-8c9d-d4d2880d7ca6"}
```
Comment (ruflin) [inline data/meta.json]: The data directory should be ignored.

Comment (dadoonet) [reply data/meta.json]: Great. I don't remember if I forgot to add it back to `.gitignore` or if it was not there when I created a blank new beats project from the template. I mean: should it be added here? https://github.com/elastic/beats/blob/master/generator/beat/.gitignore

Comment (ruflin) [reply data/meta.json]: Yes, it should probably be added here https://github.com/elastic/beats/blob/master/generator/beat/%7Bbeat%7D/.gitignore and inside the oder beat generator directories.

Comment (ruflin) [reply data/meta.json]: There is also a log directory created, I wonder if we should add this one also there.

**Pull request merged**